android/java/org/chromium/chrome/browser/crypto_wallet/util/AndroidUtils.java

```diff
@@ -89,4 +90,8 @@ private static void setViewVisibility(boolean isVisible, View... views) {
             view.setVisibility(visibility);
         }
     }
+
+    public static int getScreenHeight() {
+        return Resources.getSystem().getDisplayMetrics().heightPixels;
```

Comment:
[This version of getScreenHeight excludes all bars and `insets` ](https://stackoverflow.com/a/31377616) so please add a note here for others.